feat(iceberg): Enable alter tbl-properties to manage old metadata files (#26107)
prestodb/presto: presto-docs/src/main/sphinx/connector/iceberg.rst

    ALTER TABLE iceberg.web.page_views ADD COLUMN dt date WITH (partitioning = ARRAY['year', 'bucket(16)', 'identity']);

> Table properties can be modified for an Iceberg table using an ALTER TABLE SET PROPERTIES statement. At present, the modifiable table properties include

Reviewer: ```suggestion
Some Iceberg table properties can be modified using an ALTER TABLE SET PROPERTIES statement. The modifiable table properties are
```
Rephrasing suggestion. "Include" implies there are more modifiable properties than are listed in the sentence. 

Note: I thought rephrasing into an imperative "Use an ALTER TABLE SET PROPERTIES statement to modify some Iceberg table properties", but I feel that emphasizes the wrong thing so I recommend **not** doing that :) . I'm including my self-rejected idea in this comment to show I thought about alternatives before finding the suggestion I think works better.
